packages/vue-components/src/panels/NestedPanel.vue

```diff
@@ -358,7 +363,7 @@ export default {
         }
 
         .card-collapse > hr {
-            margin-top: 1.5rem;
+            margin-top: 1rem;
```

Comment:
```suggestion
            margin-top: 0.5rem;
```
This seems to achieve the same effect for larger screen. With 1rem:
![image](https://github.com/MarkBind/markbind/assets/41845017/5758fd07-8439-48c6-81e6-3b5cf54914b3)
But with 0.5rem:
![image](https://github.com/MarkBind/markbind/assets/41845017/c3353243-358c-42e1-b89c-92ee2ccdfd3a)